projects/extension/sql/idempotent/012-vectorizer-int.sql

```diff
@@ -301,8 +301,8 @@ begin
     where v.id operator(pg_catalog.=) vectorizer_id
     ;
 
-    -- don't let anyone but the owner of the source table call this
-    select k.relowner operator(pg_catalog.=) pg_catalog.session_user()::pg_catalog.regrole
+    -- don't let anyone but the owner (or members of the owner's role) of the source table call this
+    select pg_catalog.pg_has_role(_vectorizer_create_dependencies.caller, k.relowner, 'MEMBER')
```

Comment:
weird attribute qualification (_vectorizer_create_dependencies.caller instead of just caller), why?

Reply:
done